### PR TITLE
add user to organization + test

### PR DESCRIPTION
**Pull request by mperehinets**
add user to organization + tests

### CODE REVIEW

File: src/main/java/com/smartcity/controller/OrganizationController.java [hunk 1]
```diff
@@ -56,4 +60,11 @@ public boolean delete(@PathVariable("id") Long id) {
         return organizationService.getAll();
     }
 
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping(value = "/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
```
Comment (iholovatyi) [inline src/main/java/com/smartcity/controller/OrganizationController.java]: Use something like:
`/{organizationId}/addUser/{userId}` no need to use UserDto in request body

File: src/main/java/com/smartcity/controller/OrganizationController.java [hunk 2]
```diff
@@ -56,4 +60,11 @@ public boolean delete(@PathVariable("id") Long id) {
...
+    public boolean addUserToOrganization(@PathVariable("id") Long OrganizationId, @RequestBody UserDto userDto) {
```
Comment (iholovatyi) [inline src/main/java/com/smartcity/controller/OrganizationController.java]: Add one more action: *removeUserFromOrganization*

File: src/test/java/com/smartcity/service/OrganizationServiceImplTest.java [hunk 3]
```diff
@@ -83,4 +86,17 @@ public void testGetAllOrganizationDto() {
         doReturn(organizationList).when(organizationDao).getAll();
         assertEquals(organizationDtoList, organizationService.getAll());
     }
+
+    @Test
+    public void TestAddUserDtoToOrganizationDto() {
```
Comment (iholovatyi) [inline src/test/java/com/smartcity/service/OrganizationServiceImplTest.java]: name the method: `testAddUserToOrganization()`

Comment (mperehinets) [reply src/test/java/com/smartcity/service/OrganizationServiceImplTest.java]: ok

File: src/main/java/com/smartcity/controller/OrganizationController.java (hunk 2 above)
Comment (mperehinets) [reply src/main/java/com/smartcity/controller/OrganizationController.java]: ok

File: src/main/java/com/smartcity/controller/OrganizationController.java (hunk 1 above)
Comment (mperehinets) [reply src/main/java/com/smartcity/controller/OrganizationController.java]: ok

File: src/main/java/com/smartcity/dao/OrganizationDaoImpl.java [hunk 4]
```diff
@@ -111,6 +112,46 @@ public boolean delete(Long id) {
         }
     }
 
+    @Override
+    public boolean addUserToOrganization(Organization organization, User user) {
+        try {
+            LocalDateTime currDate = LocalDateTime.now();
+            jdbcTemplate.update(Queries.SQL_ORGANIZATION_ADD_USER_TO_ORGANIZATION, preparedStatement -> {
```
Comment (Igor-Polatajko) [inline src/main/java/com/smartcity/dao/OrganizationDaoImpl.java]: Why do you use PreparedStatement here? You can do insert without it.

**Pull request merged**